reference/release-notes/1.24.0.md

```diff
@@ -0,0 +1,101 @@
+
+* Anbox Cloud includes and uses the following latest versions for components it uses:
+  - LXC version 6.0.2<!--AC-2812-->
+  - Mesa driver version 24.2.4<!--AC-2800-->
+  - VirGL renderer version 1.9.0.<!--AC-2800-->
+
+### Security
+
+* We have enhanced security with a minimum version of TLS 1.3 as a requirement for all services working with Anbox Cloud. TLS 1.3 is also enabled by default for the Anbox Stream Agent and Gateway.<!--AC-2728 and AC-2654-->
+
+### Anbox Management Service (AMS)
+
+* We have increased the maximum number of queued operations in AMS to allow better performance.<!--AC-2831-->
+* AMS supports querying the image variant (`android` or `aaos`) for Anbox Cloud images.<!--AC-2757-->
+* The web client, for example, the Anbox Cloud dashboard can communicate with multiple AMS instances.<!--AC-2674-->
+
+### Android
+
+* This release includes Android 14 based AOSP and AAOS images.
+* We now support using the lock screen in Anbox Cloud([LP 1983426](https://bugs.launchpad.net/anbox-cloud/+bug/1983426)).
+
+### Charms
+
+* For the NFS Operator charm for Anbox Cloud, `Cachefilesd` is now optional and is used only when `fsc` is provided as a mount option.<!--AC-2839-->
+
+### Dashboard
+
+* You can now debug a session with the new developer tools on the stream page. Terminal and logs are available in parallel when streaming an instance.
+* For mobile streams, a location selector is now available.
+* For increased security, HTTPS is mandatory for all connections.
+* Image type is auto-detected once application is selected. 
+* Improved accessibility with a *Skip to content* button.
+
+### Streaming
+
+* Anbox Cloud now provides an option to allow the use of custom fragment shader-based upscaling algorithms for video streaming. This helps you to apply multiple shaders sequentially to perform multi-pass upscaling and replaces the default [AMD FidelityFX Super Resolution 1.0](https://gpuopen.com/fidelityfx-superresolution/) shader. See {ref}`sec-custom-fragment-shader` for more information.<!--AC-2541-->
+* The Anbox Streaming SDK now includes two additional examples that demonstrate shader-based upscaling algorithms integrated with the JS SDK:<!--AC-2543-->
+  - [Snapdragon™ Game Super Resolution(SGSR)](https://github.com/SnapdragonStudios/snapdragon-gsr)
+  - [Anime4K high-quality real-time anime](https://github.com/bloc97/Anime4K)
+
+### Other
+
+* Android security updates for November 2024 (see [Android Security Bulletin - November 2024](https://source.android.com/docs/security/bulletin/2024-11-01) for more information).
+* The Android WebView has been updated to 130.0.6723.73.
+
+## Removed functionality
+
+Following deprecation in earlier releases, these functionalities have been removed from Anbox Cloud with the 1.24.0 release:
+
+* Support for Ubuntu 20.04 (Focal Fossa)
+
+## Deprecations
+
+< List deprecated features. Provide an alternative, if any, for the users to use instead of the deprecated features >
```

Comment:
I wouldn't do it that way because it switches the context of the release notes which is to focus on the current release. So any deprecations we are announcing in 1.24.0 will go here. 

I have added a statement that, in addition to calling out no deprecations for 1.24.0, will remind users to look at our deprecation notices to prepare for next releases.